solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1163,17 +1171,18 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         view
         returns (Fraction memory)
     {
-        (, , uint256 averageRateN, uint256 averageRateD) = reserveTokenRates(_poolToken, _reserveToken);
+        (, , uint256 averageRateN, uint256 averageRateD) = reserveTokenRates(_poolToken, _reserveToken, true);
```

Comment:
a bit confusing that the function always passes `true` here - we can't really guarantee that it's only used in write functions (as opposed to read-only). think we should do anything about it?

Reply:
Depending on the purpose of the function.
For this specific function, you are probably right, i.e., it should take that flag as input.

Reply:
Fixed.